src/main/java/seedu/toluist/ui/UiStore.java

```diff
@@ -24,6 +26,8 @@
  * Ui will observe for changes to `shownTasks` and re-render on state change
  */
 public class UiStore {
+    public static final int INVALID_SUGGESTION_INDEX = -1;
```

Comment:
Im not sure about this cuz of my sucky english so I'll just flag this first.
Should it be INDEX_INVALID_SUGGESTION or INVALID_SUGGESTION_INDEX based on the pattern CATEGORY_DETAIL?